walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,63 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+
 
 public final class FindMeetingQuery {
+  /**Return all possible time intervals for a meeting, taking into account the optional attendees, 
```

Comment:
Nit: please format JavaDoc comments as
 /**
  * text....
  */
(unless the entire comment fits in a single line).
This makes the comment stand out better from the code, and ensures comment lines don't accidentally look like code.

Reply:
Gah. GitHub messed up my comment formatting. It should be
```
  /*
   * text
   */
```